src/kOS/Suffixed/ElementValue.cs

```diff
@@ -154,7 +152,7 @@ private static Queue<ElementPair> InitQueue(Vessel vessel, Dictionary<uint, Elem
 
         public override string ToString()
         {
-            return "ELEMENT( " + dockedVesselInfo.name + ", " + parts.Count + " )" ;
+            return string.Format("ELEMENT: {0} {1} Parts", dockedVesselInfo.name, parts.Count);
```

Comment:
in my testing i realized that the element to string had this number that was never explained. so i labeled it